frontend/public/components/dashboard/grid.tsx

```diff
@@ -59,8 +60,13 @@ export const DashboardGrid: React.FC<DashboardGridProps> = ({ mainCards, leftCar
   return <div ref={containerRef}>{grid}</div>;
 };
 
+export type GridDashboardCard = {
+  span?: DashboardCardSpan;
+  Card: React.ComponentType<any>;
```

Comment:
Nit: put `Card` above `span`, key/essential properties should go before optional/non-essential ones.

Reply:
fixed